docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: "3"
 services:
-  timeline:
+  nostalgia:
```

Comment:
Possibly silly question (but good for review, I think) -- why this change?

Reply:
it was copied from the timeline repo, but in core it should be named nostalgia :)